packages/core/src/external/aws/document-signing/document-bulk-signer-local.ts

```diff
@@ -0,0 +1,19 @@
+import { getSignedUrls } from "./bulk-sign";
+import { DocumentBulkSigner, DocumentBulkSignerRequest } from "./document-bulk-signer";
+
+export class DocumentBulkSignerLocal extends DocumentBulkSigner {
+  constructor(region: string, readonly bucketName: string, readonly apiURL: string) {
+    super(region);
+  }
+
+  async sign({ patientId, cxId, requestId }: DocumentBulkSignerRequest): Promise<void> {
+    return await getSignedUrls(
```

Comment:
nit: since it doesn't return anything (`void`) can remove the `return` cmd and only `await` it?